### PR TITLE
Moved some generic components to tacugama

### PR DESCRIPTION
**Pull request by ohalytska-ui**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "9.1.7",
+  "version": "9.1.8",
```
Comment (hiddeottens) [inline package.json:2]:
```suggestion
  "version": "9.2.0",
```

Can be marked as a minor update

**Review by hiddeottens — APPROVED** [review]
nice 🚀

**Review by OksanaTC — APPROVED** [review]

File: src/components/edit-page/EditPage.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+              testId={`cancel-button-${entity}`}
+              type="button"
+              startEnhancer={<CaretLeftIcon />}
+            >
+              {backText}
+            </Button>
+          )}
+        </ButtonBox>
+      </FlexItem>
+      <FlexGrid flexGridColumnCount={2} flexGridColumnGap="scale800" flexGridRowGap="scale1600">
```
Comment (dylanmau) [inline src/components/edit-page/EditPage.tsx:149]: Both work but maybe replace it for this for consistency
```suggestion
      <FlexGrid flexGridColumnCount={2} flexGridColumnGap={scale800} flexGridRowGap={scale1600}>
```

File: src/components/edit-page/EditPageSkeleton.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  return (
+    <>
+      <FlexItem justifyContent="space-between" marg1="0" marg2="0" marg3={scale500} marg4="0">
+        <Skeleton animation height={scale900} width="200px" />
+        <ButtonBox>
+          <Skeleton animation height={scale900} width="200px" />
+        </ButtonBox>
+      </FlexItem>
+      <FlexGrid flexGridColumnCount={2} flexGridColumnGap="scale900" flexGridRowGap="scale1600">
```
Comment (dylanmau) [inline src/components/edit-page/EditPageSkeleton.tsx:30]:
```suggestion
      <FlexGrid flexGridColumnCount={2} flexGridColumnGap={scale900} flexGridRowGap={scale1600}>
```

File: src/components/input/file-input/types.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import { MutableRefObject } from 'react';
+
+export interface FileInputProps {
+  inputRef: MutableRefObject<any>;
+  onChange: (e: any) => void;
```
Comment (dylanmau) [inline src/components/input/file-input/types.ts:5]: Think we can specify this as
```suggestion
  onChange: (e: React.FormEvent<HTMLInputElement>) => void;
```

File: src/components/multi-pages-modal/MultiPagesModal.tsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  submitText,
+  resetOnClose,
+  disableCounterTitle = false,
+  nextPageDisabled = false,
+  noSubmit = false,
+  onSubmit,
+  validatePage,
+  cancelText,
+  previousText,
+  nextText,
```
Comment (dylanmau) [inline src/components/multi-pages-modal/MultiPagesModal.tsx:22]: Maybe we can add name as optional property so you are able to pass on an unique name. I don't think we have multiPagesModal inside multiPagesModal ( as we have for regular modals ), but if we want that in the future we will need unique names to keep track which ones are open. 
```suggestion
  name = "multi-pages-modal"
```

**Review by dylanmau — CHANGES_REQUESTED** [review]
Nice refactoring work :) Comments mostly about things that were already like this in next

File: src/components/multi-pages-modal/MultiPagesModal.tsx (hunk 5 above)
Comment (ohalytska-ui) [reply src/components/multi-pages-modal/MultiPagesModal.tsx:22]: Yes, you are absolutely right

File: src/components/container/ContainerFull.tsx [hunk 6]
```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import { ContainerFullProps } from './types';
+
+export const ContainerFull = ({ children }: ContainerFullProps) => <div className="container-full">{children}</div>;
```
[the remark this answers is not in the document]
Comment (ohalytska-ui) [reply src/components/container/ContainerFull.tsx:4]: Hm, I moved those components here to import them in the fe-next from tacugama. Should we still  use here "class", I am not sure?

[the remark this answers is not in the document]
Comment (dylanmau) [reply src/components/container/ContainerFull.tsx:4]: I think we should remove the class, create an styled component inside a styled.ts like: 

```suggestion
export const StyledContainerFull = themedStyled<'div',  StyledContainerProps>('div', ({ $theme, $backgroundColor }) => ({ 
    Then put the styling from fe-next global.css here 
```
Then the component will look like: 
```suggestion
export const ContainerFull = ({ children }: ContainerFullProps) => <StyledContainerFull>{children}</StyledContainerFull>;
```
Then the styling will also be applied if we use it somewhere else then fe-next.

**Review by dylanmau — APPROVED** [review]

**Pull request merged**